recipe/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "3.2.1" %}
+{% set name = "tensorflow-datasets" %}
+{% set sha256 = "b622467b9e8ac0fdff04d07c1f3d551102e930c35d28694bbe23e8c3758ee05b" %}
+
+# This is the recipe for tensorflow-datasets
+package:
+   name: {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  noarch: python
+  script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+     - python 
+  build:
+     - git >=2.2 
+  run:
+   # conda build requirements
+     - python 
+     - absl-py {{ absl_py }}
+     - attrs {{ attrs }} 
+     - dill 
+     - future
+     - numpy
+     - promise
+     - protobuf {{ protobuf }}
+     - requests {{ requests }}
+     - six
+     - tensorflow-metadata
+     - termcolor
+     - tqdm
```

Comment:
no pinning for six, tqdm too. Is it intentional.

Reply:
No, earlier I'd kept it as it is intentionally because that's how setup.py of tensorflow/datasets mentions the dependencies. It doesn't restrict to a particular version. So, I'd done the same. But then later thought it might create issues while installation in a same conda env. So, added the pins.